### ISSUE TITLE
Add Autocomplete and ARIA Attributes to Form Fields in SignIn View

### ISSUE THREAD

**Issue by sbolel** [CMS-Enterprise/sbom-harbor-ui#82]
## Description

Currently, the SignIn view does not fully adhere to accessibility and HTML standards. Specifically, the form fields in the SignIn view lack the necessary autocomplete and ARIA (Accessible Rich Internet Applications) attributes.

The Chrome browser console flags the following error:

```
[DOM] Input elements should have autocomplete attributes (suggested: "current-password"): (More info: https://goo.gl/9p2vKq) <input aria-invalid=​"false" id=​"auth-login-v2-password" type=​"password" class=​"MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd css-1u6bhim-MuiInputBase-input-MuiOutlinedInput-input" value>​
```

This error indicates that the password input field should have an autocomplete attribute, specifically `autocomplete="current-password"`.

Additionally, the email and password fields, as well as the "Login" and "Login with IDP" buttons, lack ARIA attributes that provide essential information to assistive technologies.

## Expected Behavior

All form fields in the SignIn view should have the appropriate autocomplete and ARIA attributes. This includes:

- The email field should have an `autocomplete="email"` attribute and the necessary ARIA attributes.
- The password field should have an `autocomplete="current-password"` attribute and the necessary ARIA attributes.
- The "Login" and "Login with IDP" buttons should have the necessary ARIA attributes.

## Possible Solution

Update the SignIn view to include these attributes. For instance, the password input field can be updated as follows:

```jsx
<input
  type="password"
  autocomplete="current-password"
  aria-labelledby="labelId"
  aria-required="true"
  id="auth-login-v2-password"
  class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd css-1u6bhim-MuiInputBase-input-MuiOutlinedInput-input"
/>
```

The other fields and buttons can be updated in a similar manner.

## Additional Context

Adhering to these standards will improve the accessibility of the application and provide a better user experience for all users, including those using assistive technologies. It also helps all browsers autofill forms for users, improving the overall user experience.
